### PR TITLE
Increase version bounds for aeson

### PR DESCRIPTION
**Pull request by neilmayhew**
This change is [Reviewable](https://reviewable.io/reviews/tclem/proto-lens-jsonpb/7)

### CODE REVIEW

**Comment by tclem** [comment]
Nice, I'll take a look once the builds are green.

**Comment by neilmayhew** [comment]
I didn't notice that there was a dependency in `example` as well. I've changed this, and was able to raise both of them to `aeson ^>= 1.5.2`.

File: proto-lens-jsonpb.cabal [hunk 1]
```diff
@@ -40,7 +40,7 @@ common haskell
 common dependencies
   build-depends:
       base >=4.7 && <5
-    , aeson ^>= 1.4.5.0
+    , aeson ^>= 1.5.2
```
Comment (tclem) [inline proto-lens-jsonpb.cabal:43]:
```suggestion
    , aeson >= 1.4 && <1.6
```

File: example/example.cabal [hunk 2]
```diff
@@ -38,7 +38,7 @@ common haskell
 common dependencies
   build-depends:
       base >=4.7 && <5
-    , aeson ^>= 1.4.5.0
+    , aeson ^>= 1.5.2
```
Comment (tclem) [inline example/example.cabal:41]:
```suggestion
    , aeson >= 1.4 && <1.6
```

File: proto-lens-jsonpb.cabal (hunk 1 above)
Comment (neilmayhew) [reply proto-lens-jsonpb.cabal:43]: I agree. This was carelessness on my part.

I'd like to rebase the PR before you merge it, BTW.

Comment (neilmayhew) [reply proto-lens-jsonpb.cabal]: Sorry, I must have clicked the wrong thing. I added both suggestions to a batch and I thought I committed it, but apparently not. I've incorporated them into an updated version of the original commit.

**Pull request merged**